packages/minifiers/minifiers.js

```diff
@@ -109,7 +109,7 @@ var rewriteRules = function (rules) {
     // a folder starting with "/packages/" is served on the same path as
     // it was on the filesystem and everything else is served on root "/".
     if (! basePath.match(/^\/?packages\//i))
-        basePath = "/";
+        basePath = (process.env.ROOT_URL_PATH_PREFIX || "") + "/";
```

Comment:
I don't get how this possibly could work, since nothing seems to set that environment variable anywhere...

Can you include either (ideally) a test using the `self-test` framework in `tools/tests`, or at least manual reproduction instructions that show how this fixes the bug?